web/annotation_editor_layer_builder.css

```diff
@@ -43,6 +43,24 @@
   /*#else*/
   --editorInk-editing-cursor: url(images/cursor-editorInk.svg) 0 16, pointer;
   /*#endif*/
+  --editorHighlight-editing-cursor: url(images/cursor-editor-texthighlight.svg)
+      23.5 23,
```

Comment:
How "meaningful" is it to use a sub-pixel value? Could we perhaps use just integer values instead?